plugin/algorithm.go

```diff
@@ -0,0 +1,43 @@
+package plugin
```

Comment:
The Copyright of the Notary Project need to be added in front of the file.

Reply:
Good point, crated backlog to do this before 1.0 of this repo
https://github.com/notaryproject/notation-plugin-framework-go/issues/4